tests/Makefile.am

```diff
@@ -38,30 +40,33 @@ TESTS = referenceFAT12.mkfs              \
 
 
 XFAIL_TESTS = check-dot_entries.fsck \
-	      check-huge.fsck
+	      check-huge.fsck \
+	      check-backup_boot_sector_mismatch.fsck
```

Comment:
Cannot we fix fsck to be able to repair also this `check-backup_boot_sector_mismatch.fsck` filesystem?

Reply:
Doable, I guess. I've downgraded this to a warning: If the backup boot sector was found in one place, but the backup boot sector says the backup is in a different sector, `fsck.fat` updates the pointer in the boot sector to where the backup boot sector was actually found.

Alternately, `fsck.fat` could move the backup boot sector to where the pointer says it should be, but that feels excessive.